docs/develop/01_sdk/01_quickstart.md

```diff
@@ -61,9 +63,36 @@ For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
   </TabItem>
 </Tabs>
 
+<Tabs groupId="ts-js-choice">
+  <TabItem value='ts' label='Typescript' default>
+
+After you have imported the SDK you will be able to access the functionalities that KILT provides.
```

Comment:
Can you indent this with a tab? And the same for all the other paragraphs?

Reply:
Done